src/server/helpers/auth.js

```diff
@@ -50,46 +54,55 @@ function _updateCachedMBName(bbUserModel, mbUserJSON) {
 
 export function init(app) {
 	const {orm} = app.locals;
-	passport.use(new MusicBrainzOAuth2Strategy(
-		_.assign(
-			{
-				passReqToCallback: true,
-				scope: 'profile'
-			}, config.musicbrainz
-		),
-		async (req, accessToken, refreshToken, profile, done) => {
-			try {
-				if (req.user) {
-					const linkedUser =
-						await _linkMBAccount(orm, req.user, profile);
-
-					// Logged in, associate
-					return done(null, linkedUser.toJSON());
+	try {
+		const strategy = new MusicBrainzOAuth2Strategy(
+			_.assign(
+				{
+					passReqToCallback: true,
+					scope: 'profile'
+				}, config.musicbrainz
+			),
+			async (req, accessToken, refreshToken, profile, done) => {
+				try {
+					if (req.user) {
+						const linkedUser =
+							await _linkMBAccount(orm, req.user, profile);
+
+						// Logged in, associate
+						return done(null, linkedUser.toJSON());
+					}
+
+					// Not logged in, authenticate
+					const fetchedUser = await _getAccountByMBUserId(orm, profile);
+
+					await _updateCachedMBName(fetchedUser, profile);
+
+					return done(null, fetchedUser.toJSON());
+				}
+				catch (err) {
+					return done(null, false, profile);
 				}
-
-				// Not logged in, authenticate
-				const fetchedUser = await _getAccountByMBUserId(orm, profile);
-
-				await _updateCachedMBName(fetchedUser, profile);
-
-				return done(null, fetchedUser.toJSON());
-			}
-			catch (err) {
-				return done(null, false, profile);
 			}
-		}
-	));
+		);
+		passport.use(strategy);
 
-	passport.serializeUser((user, done) => {
-		done(null, user);
-	});
+		passport.serializeUser((user, done) => {
+			done(null, user);
+		});
 
-	passport.deserializeUser((user, done) => {
-		done(null, user);
-	});
+		passport.deserializeUser((user, done) => {
+			done(null, user);
+		});
```

Comment:
Extra tabs?

Reply:
Checked in my code editor, looks fine to me.
These lines are in a new try/catch block, so they're indented one more.